website/docs/backends/types/terraform-enterprise.html.md

```diff
@@ -27,7 +27,7 @@ Enterprise**. While this transition is in progress, you may see references to
 terraform {
   backend "atlas" {
     name         = "bigbang/example"
-    access_token = "foo"
+    ATLAS_TOKEN  = "foo"
```

Comment:
`ATLAS_TOKEN` is an environment variable, not a field in the backend configuration. So instead of being in this section, it needs to be set via `export` on the command line.
`export ATLAS_TOKEN="foo"`

Reply:
ahh